_posts/2018-01-02-bbc-newshack.html

```diff
@@ -0,0 +1,46 @@
+<p>In our demo scenario, the user gets told which article they've read last and if they'd like to leave a comment. If they choose to do so, they get presented with a question relating to the article. Due to hackathon settings, only a few questions were written for selected stories. If they answer correctly, they can record a comment which will be translated to text and posted on their behalf. Before the comment is posted, it is analysed through an API we built on the day, using the Natural Language Toolkit (NLTK) that returns the overall sentiment and how many swear words the message might contain. It gives the user time to ponder whether they want to post their comment or change it.</p>
+<p>Even if the comment contains swear words, they can go ahead and post it, but they will be charged a small fee, which we envisage going to charity (most certainly, <a href="https://www.ft.com/ft-seasonal-appeal" target="_blank">FT's Seasonal Appeal</a>).</p>
+<img src="/assets/uploads/2018/01/conversation_simulator.png" alt="FT Conversation in the DialogFlow simulator" width="50%" />
+
+<h3>Demo</h3>
+<!--TODO: upload demo video -->
+{% include youtube_video.html src="https://www.youtube.com/embed/BgA5V2GtBZM" %}
+
+<h2>Technical challenges</h2>
+<p>Given the hackathon setting and timeframe, we had to focus on a specific user journey, with some hard-coded data; although the structure is now there to put all the discussed features in place if we so wished.</p>
+<p>As part of FT Conversation, we had in mind an audio playback element, where users could choose to record a comment in their own voice. Without having explored the possibility of recording audio, playing an existing mp3 proved impossible.</p>
+<p>Finally, we need to bear in mind that users will want to edit their comments and provide them with tools to do so. Changing one word in the middle of a paragraph isn't simple just with a voice interface (could we envisage a companion app on a screen for this particular task? Or make use of the Echo Show for an Alexa version). In any case, we must ensure comments aren't automatically posted, and put in place safeguards to avoid <a href="https://www.nytimes.com/2017/10/26/reader-center/nyt-comments-section.html?_r=0" target="_blank">this kind of scenario</a>, encountered by the New York Times.</p>
+
+
+<h2>Future opportunities</h2>
+<p>FT Conversation is a proof of concept, although we currently have no plans to take it further. At least, until we can overcome the challenges mentioned previously. Putting it in place would require an extra effort of curation, to produce a handful of questions for each article, or could we automate those? That is another area to explore.</p>
+<p>Each new prototype lets us discover features in voice devices, and no doubt we'll be back with more in 2018.</p>
```

Comment:
"At least, until we can overcome the challenges mentioned previously. Putting it in place would require an extra effort of curation, to produce a handful of questions for each article, or could we automate those? That is another area to explore"

It is not at all clear that our users would want or benefit from this specific capability, but each new prototype lets us discover more possibilities and opportunities with voice devices, as well as highlighting where our own data and systems need attention.